client/lib/cart-values/cart-items.js

```diff
@@ -758,19 +758,33 @@ export function getGoogleApps( cart ) {
 	return filter( getAllCartItems( cart ), isGSuiteOrExtraLicenseOrGoogleWorkspace );
 }
 
+/**
+ * Creates a new shopping cart item for G Suite or Google Workspace.
+ *
+ * @param {object} properties - list of properties
+ * @returns {CartItemValue} the new item
+ */
 export function googleApps( properties ) {
+	const { domain, meta, product_slug, quantity, new_quantity, users } = properties;
+
+	const domainName = meta ?? domain;
+
 	const productSlug =
-		properties.product_slug ||
+		product_slug ||
 		( config.isEnabled( 'google-workspace-migration' )
 			? GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY
 			: GSUITE_BASIC_SLUG );
 
-	return assign( domainItem( productSlug, properties.meta ?? properties.domain ), {
-		extra: { google_apps_users: properties.users },
-		...( productSlug === GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY
```

Comment:
We only pass `quantity` for Google Workspace (and not G Suite), so this check is superfluous.